content/en/blog/_posts/2024-04-02-introducing-jobset/2024-04-02-introducing-jobset.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: blog
+title: "Introducing JobSet"
+date: 2024-04-02
+slug: introducing-jobset
+---
+
+**Authors**: Daniel Vega-Myhre (Google), Abdullah Gharaibeh (Google), Kevin Hannon (Red Hat)
+
+In this article, we introduce [JobSet](https://jobset.sigs.k8s.io/), an open source API for representing distributed jobs. The goal of JobSet is to provide a stable API for running/building APIs with AI/ML and HPC use cases in mind.
+
```

Comment:
Please mention that it is alpha. The goal might be stability, but “provide a stable API” might mislead people into thinking it already is.

Reply:
Lets just say "provide an API", we don't need to mention stable or alpha as it will eventually graduate.

Reply:
Updated phrasing, please let me know what you think.

Reply:
This is fine.